### ISSUE TITLE
Left/Right click of mouse mouse

### ISSUE THREAD

**Issue by SladkiyTort** [torokati44/ruffle-android#51]
Some games require the left or right click of mouse and sometime it won't work correctly can you very kind sir add the mouse buttons just like a keyboard buttons?
Btw your ruffle is greate wish you luck

**Comment by torokati44** [comment]
Sure, this is planned, I'm just not exactly sure how it should work, or when I'll have the time for it.
And thanks!

**Comment by huchao2007** [comment]
The shoThe shooting game needs the mouse to move, had better be able to support the Bluetooth mouse, thanks

**Comment by SladkiyTort** [comment]
Yeah especially divine intervention, ain't no way you will play it with fi*ger.
